src/devices/bus/interpro/sr/edge.h

```diff
@@ -0,0 +1,294 @@
+// license:BSD-3-Clause
+// copyright-holders:Patrick Mackinlay
+#ifndef MAME_BUS_INTERPRO_SR_EDGE_H
+#define MAME_BUS_INTERPRO_SR_EDGE_H
+
+#pragma once
+
+#include "sr.h"
+
+#include "screen.h"
+#include "machine/ram.h"
+#include "cpu/tms32031/tms32031.h"
+#include "video/bt45x.h"
+#include "machine/z80scc.h"
+#include "bus/interpro/keyboard/keyboard.h"
+
+enum control_mask
```

Comment:
Please don't put generically-names things in the global namespace in headers.  Either put it inside your class or get it out of the header. Putting all these generic names in a header makes trouble for anything that includes it.